content/vault/v1.21.x (rc)/content/docs/sync/gcpsm.mdx

```diff
@@ -137,7 +139,23 @@ corresponds with the planned replication policy. The key rings and keys must be
 
 When using a global KMS key, it must be the only key set on the destination and the replication locations must remain unset, meaning
 it can only be used with using GCP's automatic replication. When specifying regional keys, a key must be set for each region in the
-replication location list. GCP key names are expected in the format of the entire resource ID, e.g. `projects/<project_id>/locations/<location_name>/keyRings/<key_ring_name>/cryptoKeys/<key_name>`. See the [API](#api) section for more details.
+replication location list.
+
+GCP key names are expected in the format of the entire resource name, e.g. `projects/<project_id>/locations/<location_name>/keyRings/<key_ring_name>/cryptoKeys/<key_name>`. Be aware that this is **case-sensitive**, with special attention on the `keyRings` and `cryptoKeys` portions.
```

Comment:
```suggestion
You must specify GCP key names using the entire resource name. For example, 
`projects/<project_id>/locations/<location_name>/keyRings/<key_ring_name>/cryptoKeys/<key_name>`.
GCP key names, including the `keyRings` and `cryptoKeys` segments, are
case-sensitive.
```
Style correction: avoid "this" as a pronoun, avoid using bold strictly for emphasis